README.md

```diff
@@ -198,14 +198,139 @@ Example:
 }
 ```
 
+### Filtering Content Items
+
+**Note:** Filtering content via `filterBy() | filterByContentType() | filterByParentId() | filterContentTypes()` is only supported when using [Content Delivery 2](#content-delivery-versions)
+
+Once you have [a schema set up for filtering](https://amplience.com/docs/development/contentdelivery/filterandsort.html), the content item must be published before it can be retrieved using this SDK.
```

Comment:
Suggestion instead of the above.

> Filtering by Content Type or Parent ID is enabled by default. You can also filter by any other field in your schema once [you enable it](https://amplience.com/docs/development/contentdelivery/filterandsort.html).
 
I don't think we need the note about publishing? I understand why people not realise they need to publish when changing a delivery key, but content needing publishing seems obvious to me.